===================================================================
RCS file: /srv/anoncvs.freedesktop.org/cvs/xorg/xserver/xorg/randr/Makefile.am,v
rcsdiff: /srv/anoncvs.freedesktop.org/cvs/xorg/xserver/xorg/randr/Makefile.am,v: warning: Unknown phrases like `commitid ...;' are present.
retrieving revision 1.4
retrieving revision 1.5
diff -u -r1.4 -r1.5
--- xorg/randr/Makefile.am	2005/12/02 06:02:44	1.4
+++ xorg/randr/Makefile.am	2006/01/19 14:51:09	1.5
@@ -2,6 +2,8 @@
 
 AM_CFLAGS = $(DIX_CFLAGS) @SERVER_DEFINES@ @LOADER_DEFINES@
 
+sdk_HEADERS = randrstr.h
+
 librandr_la_SOURCES =	\
 	mirandr.c	\
 	randr.c		\
